yt/geometry/selection_routines.pyx

```diff
@@ -1076,11 +1076,7 @@ cdef class RegionSelector(SelectorObject):
         cdef np.float64_t r2 = radius**2
         cdef np.float64_t dmin = 0
         for i in range(3):
-            if self.periodicity[i] and self.check_period[i]:
```

Comment:
I now see that what I did here is wrong but I am not sure how to fix it. Anyone have any ideas? ping @matthewturk @AshKelly

Reply:
This stuff always confuses me.  Would it be OK to go back to the original version, and punt on this for a later commit?  Because I *think* it should mostly still work ... right?

Reply:
@matthewturk no, unfortunately the original way doesn't work correctly either. I think we have to resolve it now.

Reply:
Never mind, I figured this out!!